test/net/sourceforge/kolmafia/DataFileConsistencyTest.java

```diff
@@ -185,7 +185,8 @@ public void testFamiliarHatchlings() throws IOException {
       if (ItemDatabase.getConsumptionType(id) == ConsumptionType.FAMILIAR_HATCHLING) {
         var name = ItemDatabase.getItemDataName(id);
         // replica familiars from Legacy of Loathing are special
-        if (!name.startsWith("replica ")) {
+        // wet paper tiger familiar is not public at the moment
```

Comment:
When is that expected to change?  Perhaps delay this until it is public?

Reply:
I get the impression it's going to be a while.